### DISCUSSION TITLE
Does Clarity Design System project gets impacted somehow after Broadcom acquisition?

### DISCUSSION THREAD

**Discussion by arale61** [vmware-clarity/core#284] [Q&A] [answered]
Hello,
I will like to know if Clarity Design System projects is impacted after Broadcom acquisition?
Please, extend as much as possible on details to enable us understand the current situation. Is very interesting for those we are using Clarity as a dependency for our products.
Thanks in advance.

**Comment by kristinaross** [comment, marked as answer]
Hello @arale61,

Broadcom is aware that there are external, commercial teams who use Clarity Design System to rapidly build solutions that family well with VMware products.

The Clarity Design System is continuing as an integral part of the VMware Cloud Foundation Division in Broadcom. We plan to continue to publish to open source.

Any significant changes to open source availability will be announced in advance through this and other Clarity-related channels on Github.

**Reply by arale61** [reply]
Thanks a lot for this valuable information and quick response.

**Reply by TekSiDoT** [reply]
As an outside observer, Clarity Design System (or at least the Open Source part visible from the outside), has been in a sorry state for the past 2-3 years.

* There is no roadmap for new features
* Features that have been promised and delayed again and again in the name of accesibility are still nowhere in sight
* The Core/Angular split has led to two understaffed projects
* Important Design related work has not been updated (The Figma lib saw its last update 3 years ago)
* References to internal channels and emails have increased.

Personally, I wouldn't expect anything beyond the most basic bug fixes and further team attrition. Feel free to convince me otherwise.

**Reply by kristinaross** [reply]
Hello @TekSiDoT --

Thank you for your feedback. We do hear you, and are aware that there are areas in need of refresh or new attention. 

As mentioned above, Clarity Design System is continuing as an integral part of the VMware Cloud Foundation Division. Major acquisitions take time to propose, plan, and complete, and such transactions do impact the teams, priorities, and roadmaps of all within the acquired company. 

In fact, Clarity now has a larger team, balanced between a solid core of returning veterans and the addition of new contributors, internally. We are actively defining our roadmaps for beyond the v17 Clarity NG release. 

Clarity NG v17 (releasing this week) completes the process of introducing design tokens. Design tokens provide necessary infrastructure for the future evolution of Clarity's visual language and themes, and help us align the visual language across both NG and Core, too.

We will share more information about the roadmap and related plans for Open Source when we can.

**Reply by TekSiDoT** [reply]
Hi @kristinaross ,

Thank you for your response. I'm looking forward to seeing these organizational improvements impact the projects health. You should consider writing a (blog) post about the project and its future.

**Reply by d-m-s** [reply]
I share the observations of @TekSiDoT
In the recent v17 release notes an update to the Figma lib was announced ... 

@kristinaross is this just internally available? See also:
https://github.com/vmware-clarity/ng-clarity/discussions/1303#discussion-6325141

**Reply by kevinbuhmann** [reply]
The v17 Figma library is currently only available internally.

**Reply by TekSiDoT** [reply]
@kristinaross Another 6 months have passed with no communication - with the notable exception of @kevinbuhmann doing his best to keep this project going.

Is there *any* plan for proceeding with the project in open source space, or should we expect the clarity.design page to be unavailable without further explanation from one day to another (like, for example, the figma libraries previously available at https://www.figma.com/@vmware and still linked to from your projects main page).

I loved Clarity UI, I loved building stuff with it, but now it seems it's time to migrate a whole bunch of apps.

**Reply by kevinbuhmann** [reply]
I am sorry about the Figma link being completely unavailable now. Broadcom disabled all passwordless Figma links. We didn't get much notice before the change was made. The public Figma library was already outdated, and I'm not sure about the status of making the updated Figma library publicly available or if it would even be allowed given the Figma policy change.

>Is there any plan for proceeding with the project in open source space?

If I was building a non-VCF (VMware Cloud Foundation) app, I would not use Clarity.

**Reply by TekSiDoT** [reply]
Thank you, and good luck!

**Comment by squidjam** [comment]
> Hello @TekSiDoT --
> 
> Thank you for your feedback. We do hear you, and are aware that there are areas in need of refresh or new attention.
> 
> As mentioned above, Clarity Design System is continuing as an integral part of the VMware Cloud Foundation Division. Major acquisitions take time to propose, plan, and complete, and such transactions do impact the teams, priorities, and roadmaps of all within the acquired company.
> 
> In fact, Clarity now has a larger team, balanced between a solid core of returning veterans and the addition of new contributors, internally. We are actively defining our roadmaps for beyond the v17 Clarity NG release.
> 
> Clarity NG v17 (releasing this week) completes the process of introducing design tokens. Design tokens provide necessary infrastructure for the future evolution of Clarity's visual language and themes, and help us align the visual language across both NG and Core, too.
> 
> We will share more information about the roadmap and related plans for Open Source when we can.

hi @kristinaross, 
Thanks for your response, but you will understand how confusing it is that issues and discussions go without responses of any kind for years.
I really hope communications starts to improve and that `core` starts getting the same kind of love as `ng-clarity`.
Best wishes.

**Reply by kristinaross** [reply]
I do understand, @squidjam -- thank you for commenting.
